ERROR at //build/config/c++/modules.gni:43:7: Unknown function.
      path_exists("//build/modules/${module_platform}/BUILD.gn"),
      ^----------
See //BUILD.gn:11:1: whence it was imported.

--- a/build/config/c++/modules.gni
+++ b/build/config/c++/modules.gni
@@ -39,10 +39,6 @@ if (is_chromeos) {
 if (clang_modules_platform_supported && use_autogenerated_modules) {
   chrome_build_file_bug = "https://g-issues.chromium.org/issues/new?cc=chrome-build-team@google.com&component=1456832&type=BUG&priority=P2&severity=S2&template=1959503"
 
-  assert(
-      path_exists("//build/modules/${module_platform}/BUILD.gn"),
-      "Please file a bug at ${chrome_build_file_bug} containing your " +
-          "args.gn, and mention that it failed on ${current_os}-${current_cpu}")
 }
 
 if (!use_autogenerated_modules && use_clang_modules) {
